firefox_puppeteer/ui/menu.py

```diff
@@ -70,3 +95,19 @@ def select(self, label):
                 raise NoSuchElementException(message)
 
             return item[0].click()
+
+        def select_by_id(self, menu_item_id):
+            """Click on a menu item within this menu.
+
+            :param menu_item_id: The ID of the menu item, e.g.,
+            **menu_newNavigatorTab**.
+            """
+            item = [l for l in self.items if l.get_attribute('id') ==
+                    menu_item_id]
```

Comment:
I don't think it will handle multi-level menus like `View -> Toolbars -> Menu Bar`. We would need a recursive strategy to go through each menu hierarchy.